tests/Kernel/Paragraphs/ParagraphsTest.php

```diff
@@ -635,4 +636,85 @@ public function testFactsFigures(): void {
     $this->assertCount(1, $crawler->filter('div.ecl-fact-figures.ecl-fact-figures--col-3 div.ecl-fact-figures__items'));
   }
 
+  /**
+   * Test 'Description List' paragraph rendering.
+   */
+  public function testDescriptionList(): void {
+    // Remove the auto-paragraph and url-to-link filters from the plain text.
+    /** @var \Drupal\filter\Entity\FilterFormat $format */
+    $format = FilterFormat::load('plain_text');
+    $format->filters();
+    $format->removeFilter('filter_url');
+    $format->removeFilter('filter_autop');
+    $format->save();
+
+    FilterFormat::create([
+      'format' => 'filtered_html',
+      'name' => 'Filtered HTML',
+      'filters' => [
+        'filter_html' => [
+          'status' => 1,
+          'settings' => [
+            'allowed_html' => '<strong>',
+          ],
+        ],
+      ],
+    ])->save();
+
+    FilterFormat::create([
+      'format' => 'full_html',
+      'name' => 'Full HTML',
+    ])->save();
+
+    $paragraph = Paragraph::create([
+      'type' => 'oe_description_list',
+      'field_oe_title' => 'Overview',
+      'field_oe_description_list_items' => [
+        [
+          'term' => 'Term 1',
+          'description' => 'Description 1',
+        ],
+        [
+          'term' => 'Term 2',
+          'description' => '<p>Description 2</p>',
+        ],
+        [
+          'term' => 'Term 3',
+          'description' => '<p>Description <strong>3</strong></p>',
+          'format' => 'plain_text',
+        ],
+        [
+          'term' => 'Term 4',
+          'description' => '<p>Description <strong>4</strong></p>',
+          'format' => 'filtered_html',
+        ],
+        [
+          'term' => 'Term 5',
+          'description' => '<p>Description <strong>5</strong></p>',
+          'format' => 'full_html',
+        ],
+      ],
+    ]);
+
+    $paragraph->save();
+    $html = $this->renderParagraph($paragraph);
+    $crawler = new Crawler($html);
+
+    $this->assertCount(1, ($crawler->filter('h2.ecl-u-type-heading-2')));
+    $this->assertEquals('Overview', trim($crawler->filter('h2.ecl-u-type-heading-2')->text()));
+    $this->assertCount(1, $crawler->filter('dl.ecl-description-list.ecl-description-list--horizontal.ecl-description-list--featured'));
+    $this->assertCount(5, $crawler->filter('dt.ecl-description-list__term'));
```

Comment:
We cannot use the FieldListAssert because we need to get the formatted description for each item.
This assertion for example https://github.com/openeuropa/oe_theme/pull/733/files#diff-20f9f6ae60aacc67f4bc0dbe305153caa31575ccc3256873e83597b1c69b3778R711 would be '<p>Description 2</p>' in the pattern assert class.

Reply:
we discussed we will improve in a followup for checking markup instead of plain text